src/tests/functional/shared_test_classes/include/shared_test_classes/base/benchmark.hpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cctype>
+#include <iostream>
+
+#include "layer_test_utils.hpp"
+#include "ov_subgraph.hpp"
+
+namespace LayerTestsDefinitions {
+
+template <typename BaseLayerTest>
+class BenchmarkLayerTest : public BaseLayerTest {
+    static_assert(std::is_base_of<LayerTestsUtils::LayerTestsCommon, BaseLayerTest>::value,
+                  "BaseLayerTest should inherit from LayerTestsUtils::LayerTestsCommon");
+
+public:
+    static constexpr int kDefaultNumberOfAttempts = 100;
+
+    void RunBenchmark(const std::initializer_list<std::string>& node_type_names,
+                      const std::chrono::milliseconds warmupTime = std::chrono::milliseconds(2000),
+                      const int numAttempts = kDefaultNumberOfAttempts) {
+        bench_node_type_names_ = node_type_names;
+        warmup_time_ = warmupTime;
+        num_attempts_ = numAttempts;
+        LayerTestsUtils::LayerTestsCommon::configuration = {{"PERF_COUNT", "YES"}};
+        LayerTestsUtils::LayerTestsCommon::Run();
+    }
+
+    void RunBenchmark(const std::string& name,
+                      const std::chrono::milliseconds warmupTime = std::chrono::milliseconds(2000),
+                      const int numAttempts = kDefaultNumberOfAttempts) {
+        if (!name.empty()) {
```

Comment:
`RunBenchmark(name.is_empty() ? {} : {name}, warmupTime, numAttempts);`

Reply:
It would not be that short:
```cpp
RunBenchmark((nodeTypeName.empty() ? std::initializer_list<std::string>{} : std::initializer_list<std::string>{name}), warmupTime, numAttempts);
```
or
```cpp
RunBenchmark((nodeTypeName.empty() ? std::vector<std::string>{} : std::vector<std::string>{name}), warmupTime, numAttempts);
```